hub/instruction_handling/create/peeking_flag_shorthands.tex

```diff
@@ -22,6 +22,16 @@
 				+ & \peekContext      _{i + \createExceptionCallerContextRowOffset} \\
 			\end{array} \right]
 		\]
+	\item[\underline{Exception: \maxcsxSH{}:}]
+		we define
+		\[
+			\peekingSumMaxcsx
+			\define
+			\left[ \begin{array}{ll}
+				+ & \peekingSumStandardPrefix  \\
+				+ & \peekContext      _{i + \createExceptionCallerContextRowOffset} \\
+			\end{array} \right]
+		\]
```

Comment:
We don't break the mould for exceptional `CREATE`'s. It is inefficient (one does not need the current context row) but it simplifies everything not to differentiate.